munigeo/importer/turku.py

```
@@ -0,0 +1,641 @@
+        muni_dict = {}
+        for muni in Municipality.objects.all():
+            muni_dict[muni.name] = muni
+
+        for srv_id in list(SERVICE_CATEGORY_MAP.keys()):
+            cat_type, cat_desc = SERVICE_CATEGORY_MAP[srv_id]
+            cat, c = POICategory.objects.get_or_create(
+                type=cat_type, defaults={"description": cat_desc}
+            )
+
+            self.logger.info("Importing %s" % cat_type)
+            ret = requests.get(
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)